packages/workspace/src/utils/default-base.spec.ts

```diff
@@ -0,0 +1,57 @@
+import * as cp from 'child_process';
```

Comment:
Please move this to `packages/workspace/src/utilities`. `src/utils` are not supposed to be used because they might contain `@angular-devkit` which this does not.

Reply:
That makes sense, I didn't know the split was like that.